modules/katamari/src/test/ts/atomic/api/struct/AdtTest.ts

```diff
@@ -143,7 +143,7 @@ describe('atomic.katamari.api.struct.AdtTest', () => {
         const branches = Arr.mapToObject(original, () => Fun.identity);
         subject.match(branches);
         return false;
-      } catch (err) {
+      } catch (err: any) {
```

Comment:
I guess there is not much point in spending time to add proper types for the the tests for the Adt stuff since it's so legacy by now. Since wouldn't this be `Error` or something?

Reply:
Strict mode requires this to be `any` or `unknown` and then you need to add type guards or casts to handle the error (some details are in https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables). So since it's just in tests I didn't see it as worthwhile to add that 🤷.

Reply:
For reference, this is the error you get if you try to use anything other than `any` or `unknown`:
```
 error TS1196: Catch clause variable type annotation must be 'any' or 'unknown' if specified.
```

Reply:
Right makes sense in a language without exception tracking like Java. It could be other things than error instances being thrown.